activiti-cloud-services-runtime-bundle/activiti-cloud-services-events/pom.xml

```diff
@@ -99,6 +99,10 @@
       <groupId>org.springframework.cloud</groupId>
       <artifactId>spring-cloud-stream</artifactId>
     </dependency>
+    <dependency>
```

Comment:
I don't see this being used in any of the classes in the PR. Was it just missing already or is it used in the background somehow?

Reply:
ITs passed too actually

Reply:
Good spot @ryandawsonuk, it was initially used by a piece of code that I refactored later on and it's no longer used. I'll remove it.